cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,34 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.1.1 <http://docs.cask.co/cdap/3.1.1/index.html>`__
+=============================================================
+
+Bug Fixes
+---------
+
+- `CDAP-3259 <https://issues.cask.co/browse/CDAP-3259>`__ -
+  Removed a "live re-load" script accidentally included in the previous release.
+
+- `CDAP-3321 <https://issues.cask.co/browse/CDAP-3321>`__ -
+  Fixed a problem of being unable to enable SSL on the CDAP-UI.
+
+- `CDAP-3340 <https://issues.cask.co/browse/CDAP-3340>`__ -
+  Fixed a problem with the deployment of applications using the CDAP CLI on Windows.
```

Comment:
...applications and batch loading of events to a stream using the CDAP CLI on Windows.